### DISCUSSION TITLE
Can I build without output: 'export' ?

### DISCUSSION THREAD

**Discussion by notwatermango** [saltyshiomix/nextron#504] [Q&A]
It was mentioned here.
```js
// in `./renderer/next.config.js`
module.exports = {
  // we need to export static files so as Electron can handle them
  output: 'export',
```

Can i just use output: 'standalone'? I only need electron to run my app in `webSecurity: false`

But when I run build there is no dist folder.

**Comment by BrightXiaoHan** [comment]
any solution here?
